org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss

```diff
@@ -20,13 +21,41 @@
     background-color: #FFF;
     padding: 10px 0 !important;
 
+    .crm-contact_activities-list{
+
+    }
     .crm-contact_actions-list {
       border-color: $crm-background !important;
+      width: 175px;
+      margin-left: 175px;
+      margin-right: 175px;
+      ul {
+        .crm-action-new-case {
+          display: none;
+        }
+        .crm-action-new-relationship {
+          display: none;
+        }
+        .crm-action-new-note {
+          display: none;
+        }
+        .crm-action-tag-contact {
```

Comment:
why don't you just group all the selectors? You are applying the same style for them
```scss
.selector1,
.selector2,
.selector3 {
  display: none;
}
```